wpilibj/src/main/java/edu/wpi/first/wpilibj/smartdashboard/FieldObject2d.java

```diff
@@ -125,7 +122,7 @@ synchronized void updateEntry(boolean setDefault) {
     int ndx = 0;
     for (Pose2d pose : m_poses) {
       Translation2d translation = pose.getTranslation();
-      arr[ndx + 0] = translation.getX();
+      arr[ndx] = translation.getX();
```

Comment:
The `+ 0` is a style thing to match the pattern of assignments in this function.